src/types/ArrayType.ts

```diff
@@ -1,13 +1,13 @@
-import { BrsType } from './BrsType';
```

Comment:
Only changes for the `types/` folder is that I renamed the bsc interface `BrsType` to `BscType`